TLSharp.Core/Network/MtProtoSender.cs

```diff
@@ -214,39 +253,68 @@ private bool processMessage(ulong messageId, int sequence, BinaryReader messageR
                     return HandleRpcResult(messageId, sequence, messageReader, request);
                 case 0x3072cfa1: // gzip_packed
                                  //logger.debug("MSG gzip_packed");
-                    return HandleGzipPacked(messageId, sequence, messageReader, request, token);
+                    return await HandleGzipPackedAsync(messageId, sequence, messageReader, request, token);
                 case 0xe317af7e:
                 case 0xd3f45784:
                 case 0x2b2fbd4e:
                 case 0x78d4dec1:
                 case 0x725b04c3:
                 case 0x74ae4240:
-                    return HandleUpdate(messageId, sequence, messageReader);
+                case 0x11f1331c:
+                    return HandleUpdate(code, sequence, messageReader, request);
                 default:
-                    //logger.debug("unknown message: {0}", code);
                     return false;
             }
         }
 
-        private bool HandleUpdate(ulong messageId, int sequence, BinaryReader messageReader)
+        private bool HandleUpdate(uint code, int sequence, BinaryReader messageReader, TLMethod request)
         {
-            return false;
+            try
+            {
+                var update = ParseUpdate(code, messageReader);
+                if (update != null && UpdatesEvent != null)
+                {
+                    UpdatesEvent(update);
+                }
+                return true;
+            }
+            catch (Exception e)
```

Comment:
why this catch?

Reply:
not sure, will remove so we see any exception bubble up